config/tests/testgrids/config_test.go

```diff
@@ -356,6 +356,7 @@ var noPresubmitsInTestgridPrefixes = []string{
 	"kubernetes-sigs/cluster-capacity",
 	"kubernetes-sigs/gcp-filestore-csi-driver",
 	"kubernetes-sigs/kind",
+	"kubernetes-sigs/kubetest2",
```

Comment:
+1 to https://github.com/kubernetes/test-infra/issues/18159#issuecomment-663402819

having a testgrid dashboard for non e2e presubmits just adds noise.

Reply:
testgrid also can't really keep up with k/k where it might be useful, prow.k8s.io / deck is good enough for presubmits (along with go.k8s.io/triage)